### PR TITLE
feat: support array fields like int[]

### PR DESCRIPTION
**Pull request by pascalgn**
I hope you're not scared of all the PRs :)

I'm in the middle of trying to use javaflow with our (quite large) code base, so there are some minor things I need to fix. I try to keep the PRs separately so you can decide on each one separately!

### CODE REVIEW

File: src/main/java/com/github/havardh/javaflow/phases/parser/java/JavaParser.java [hunk 1]
```diff
@@ -67,16 +67,12 @@
 
   private static boolean containsEnum(CompilationUnit cu) {
     return cu.getTypes().stream()
-        .filter(t -> t instanceof EnumDeclaration)
-        .findFirst()
-        .isPresent();
+        .anyMatch(t -> t instanceof EnumDeclaration);
```
Comment (pascalgn) [inline src/main/java/com/github/havardh/javaflow/phases/parser/java/JavaParser.java]: This is just cosmetics (it's a bit shorter), but I can remove it again if you don't like it of course.

**Comment by havardh** [comment]
Ah, this is greatly appreciated :) As long as you add features which come natural, like the ones you've already opened, I'll merge them without much friction :) Feel free to open issues for discussion if you are considering anything major. 

I'm not scared as long as you don't get discouraged by having to wait a bit for feedback/merges. ;)

File: src/test/java/com/github/havardh/javaflow/JavaFlowTest.java [hunk 2]
```diff
@@ -191,7 +191,9 @@ public void shouldMapString() {
         public void shouldMapCharArrayToString() {
```
Comment (havardh) [inline src/test/java/com/github/havardh/javaflow/JavaFlowTest.java]: We should make this two distinct tests, as the original mapped a `char[]` into a string type, while the added cases operates on arrays :) So lets add `shouldMapGenericArrays`. We could also rename the old test model to `ModelWithCharArray`.

Comment (pascalgn) [reply src/test/java/com/github/havardh/javaflow/JavaFlowTest.java]: Fixed!

**Pull request merged**